tower-balance/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![deny(rust_2018_idioms)]
+#![allow(elided_lifetimes_in_paths)]
```

Comment:
Oh, it's because of the above `deny`. Do we need it?

Reply:
It seems good to enforce style we care about?

Reply:
Eh, I don't care either way.